src/components/project_admin/suite_settings.vue

```diff
@@ -50,19 +48,52 @@
           </template>
         </select-object>
       </div>
-
       <div class="toggle-container form-field-wrapper">
-        <toggle v-model="d_suite.allow_network_access"
-                @input="$emit('field_change', d_suite)"
-                ref="allow_network_access">
+        <div>
+          <i class="fas fa-exclamation-triangle warning-tip"></i>
+          <tooltip width="large" placement="right">
+            Allowing network access may expose your files to students' code.
+          </tooltip>
+        </div>
+        <toggle :value="d_suite.allow_network_access"
+                @input="on_toogle_allow_network"
+                ref="allow_network_access"
+                >
           <div slot="on">
             Allow network access
           </div>
           <div slot="off">
             Block network access
           </div>
         </toggle>
+        <modal v-if="d_show_allow_network_access_modal"
+           @close="d_show_allow_network_access_modal = false"
+           ref="allow_network_access_modal"
+           size="large"
+           :click_outside_to_close="true"
+           :show_closing_x="true">
+            <div class="modal-header">
+              Confirm Access
+            </div>
+            <div class="modal-body">
+              Are you sure you want to allow network access in this sandbox?
+                <br>
+              This may trivially expose your files to students' code.
```

Comment:
(Same message as suggested above)

Allowing network access means that student code could steal sensitive files or other information present in the sandbox (e.g., test cases, solution files) with a simple network request. If network access is required for your test cases to work, you may be able to mitigate the risk by deleting sensitive files at the and of the test suite setup phase. If there are sensitive files that cannot be deleted from the sandbox, then you should be aware of the risks associated with this setting.